### ISSUE TITLE
can we do something about the pull requests not always modifying generated documentation?

### ISSUE THREAD

**Issue by mircealungu** [mircealungu/practice-as-a-service#91]
can we remove from version control the generated files?

i see hundreds of files changed and  thousands of lines of code even for a small pull request. 

https://www.dropbox.com/s/szwplwj6rvhsdx0/Screenshot%202017-06-14%2023.16.21.png?dl=0

**Comment by martinavagyan** [comment]
I agree the generated files are messing with the useful information, my plan was to do a separate pull request for documentation generation so that you won't need to check them. Another option is to use gh-pages and not have them on the master branch docs folder.

**Comment by mircealungu** [comment]
gh-pages would be great!
